submissions/kovalov/js-pre-oop/index.js

```diff
@@ -0,0 +1,46 @@
+const dog = {
+  species: 'dog',
+  name: 'Patron',
+  gender: 'male',
+  legs: 4,
+  hands: 0,
+  saying: 'woof',
+};
+
+const cat = {
+  species: 'cat',
+  name: 'Murzyk',
+  gender: 'male',
+  legs: 4,
+  hands: 0,
+  saying: 'meow',
+};
+
+const woman = {
+  species: 'human',
+  name: 'Anna',
+  gender: 'female',
+  legs: 2,
+  hands: 2,
+  saying: 'hello',
+};
+
+const man = {
+  species: 'human',
+  name: 'Joey',
+  gender: 'male',
+  legs: 2,
+  hands: 2,
+  saying: 'How you doin?',
+};
+
+const catWoman = Object.create(cat);
+catWoman.name = 'Cat-woman';
+catWoman.species = 'human';
+catWoman.gender = 'female';
+catWoman.legs = 2;
+catWoman.hands = 2;
+
+const inhabitants = [dog, cat, woman, man, catWoman];
+
+inhabitants.forEach((item) => print(Object.values(item).join('; ')));
```

Comment:
Well done.

Note that (a) `Object` methods doesn't guarantee a particular order of keys or values and (b) other contributors to your project may define new objects' props in a different order (and you may miss this during code review), also (c) project may require adding "private" properties to the objects, so the recommended approach is to list required properties explicitly in  use cases like this one.

It's not a hard requirement in this particular task but you may want to accept this small challenge to develop a proper habit.

Reply:
Updated. Please review.